3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -18,18 +18,60 @@ Complete the four functions provided in the starter `index.js` file:
 Try and avoid using global variables. Instead, use function parameters and 
 return values to pass data back and forth.
 ------------------------------------------------------------------------------*/
-function fetchData(/* TODO parameter(s) go here */) {
-  // TODO complete this function
-}
 
-function fetchAndPopulatePokemons(/* TODO parameter(s) go here */) {
+async function fetchData(url) {
   // TODO complete this function
+  const response = await fetch(url);
+  const json = await response.json();
+  return json;
 }
 
-function fetchImage(/* TODO parameter(s) go here */) {
+function fetchAndPopulatePokemons(pokemons, select) {
```

Comment:
You are not passing `pokemons` here, you are passing the `results` property which contains the pokemons.